FABulous/fabric_definition/Fabric.py

```
@@ -252,10 +260,16 @@ def __repr__(self) -> str:
         return fabric
 
     def getTileByName(self, name: str) -> Tile | None:
-        return self.tileDic.get(name)
+        ret = self.tileDic.get(name)
+        if ret is None:
+            ret = self.unusedTileDic.get(name)
+        return ret
```

Instead of returning `None`, I think we should raise an error.